app/javascript/dashboard/i18n/locale/en/mfa.json

```diff
@@ -0,0 +1,105 @@
+      "REGENERATE": "Regenerate Backup Codes",
+      "DISABLE_MFA": "Disable 2FA",
+      "DISABLE_MFA_DESC": "Remove two-factor authentication from your account",
+      "DISABLE_BUTTON": "Disable Two-Factor Authentication"
+    },
+    "DISABLE": {
+      "TITLE": "Disable Two-Factor Authentication",
+      "DESCRIPTION": "You'll need to enter your password and a verification code to disable two-factor authentication.",
+      "PASSWORD": "Password",
+      "OTP_CODE": "Verification Code",
+      "OTP_CODE_PLACEHOLDER": "000000",
+      "CONFIRM": "Disable 2FA",
+      "CANCEL": "Cancel",
+      "SUCCESS": "Two-factor authentication has been disabled",
+      "ERROR": "Failed to disable MFA. Please check your credentials."
+    },
+    "REGENERATE": {
+      "TITLE": "Regenerate Backup Codes",
+      "DESCRIPTION": "This will invalidate your existing backup codes and generate new ones. Enter your verification code to continue.",
+      "OTP_CODE": "Verification Code",
+      "OTP_CODE_PLACEHOLDER": "000000",
+      "CONFIRM": "Generate New Codes",
+      "CANCEL": "Cancel",
+      "NEW_CODES_TITLE": "New Backup Codes Generated",
+      "NEW_CODES_DESC": "Your old backup codes have been invalidated. Save these new codes in a secure location.",
+      "CODES_IMPORTANT": "Important:",
+      "CODES_IMPORTANT_NOTE": " Each code can only be used once. Save them before closing this window.",
+      "DOWNLOAD_CODES": "Download Codes",
+      "COPY_ALL_CODES": "Copy All Codes",
+      "CODES_SAVED": "I've Saved My Codes",
+      "SUCCESS": "New backup codes have been generated",
+      "ERROR": "Failed to regenerate backup codes"
+    }
+  },
+  "MFA_VERIFICATION": {
+    "TITLE": "Two-Factor Authentication",
+    "DESCRIPTION": "Enter your verification code to continue",
+    "AUTHENTICATOR_APP": "Authenticator App",
+    "BACKUP_CODE": "Backup Code",
+    "ENTER_OTP_CODE": "Enter 6-digit code from your authenticator app",
+    "ENTER_BACKUP_CODE": "Enter one of your backup codes",
+    "BACKUP_CODE_PLACEHOLDER": "000000",
+    "VERIFY_BUTTON": "Verify",
+    "TRY_ANOTHER_METHOD": "Try another verification method",
+    "CANCEL_LOGIN": "Cancel and return to login",
+    "HELP_TEXT": "Having trouble signing in?",
+    "LEARN_MORE": "Learn more about 2FA",
+    "HELP_MODAL": {
+      "TITLE": "Two-Factor Authentication Help",
+      "AUTHENTICATOR_TITLE": "Using an Authenticator App",
+      "AUTHENTICATOR_DESC": "Open your authenticator app (Google Authenticator, Authy, etc.) and enter the 6-digit code shown for your Chatwoot account.",
+      "BACKUP_TITLE": "Using a Backup Code",
+      "BACKUP_DESC": "If you don't have access to your authenticator app, you can use one of the backup codes you saved when setting up 2FA. Each code can only be used once.",
+      "CONTACT_TITLE": "Need More Help?",
+      "CONTACT_DESC": "If you've lost access to both your authenticator app and backup codes, please contact your administrator for assistance."
```

Comment:
If cloud ask them to reach out to Chatwoot support, if self-hosted reach out to administrator.

@tds-1 Please check `ENTERPRISE_PAYWALL` implementation to see how you can switch between text based on whether the deployment is cloud or not.

Reply:
Done https://github.com/chatwoot/chatwoot/pull/12372/commits/cf2aec9be6f9d8f310655495768fd6d2e9468044